src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+$main-color: #060b35;
+$secondary-color: #616070;
+$accent-color: #00acdc;
+
+body {
+  margin: 0;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+}
+
+h2,
+h3,
+p {
+  margin: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  min-height: 406px;
```

Comment:
do not add `height` or `min-height` 
![image](https://github.com/mate-academy/layout_product-cards/assets/22824431/973c9c89-089b-422f-b461-b11561ed7d34)